ios/ios_project_setup.md

```diff
@@ -0,0 +1,61 @@
+# How To Create A New Project
+## Create Xcode project and Github repo
+1. Create repo on Github with `.gitignore` and `README.md`. Usually, repos should be named with the format `<project-name>-ios` e.g `bose-stetson-ios`
+
+1. Create Xcode Project
+
+1. In the Xcode project directory, run:
+    ```
+    git init
+    git remote add origin <remote-url>
+    git pull origin master
+    git push --set-upstream origin master
+    ```
+    > Pull before pushing to get the Github-generated `.gitignore` so ignored files don't end up in the repo
+
+1. Update `README.md` to use [Intrepid README template](https://github.com/IntrepidPursuits/sherpa/blob/master/readme-template)
+1. Add and commit, then push to `master`
+
+## Add Cocoapods
+1. In project directory, run:
+    ```
+    pod init
+    pod install
+    ```
+    > Use `.xcworkspace` instead of `.xcodeproj` for this project from now on
+
+1. Decide whether Pods should be checked in or not. The automatically generated Podfile is set up to check them in. If they shouldn't be checked in, uncomment the `Pods` line in the Podfile.
+    > `# Pods/` -- Pods checked in
+    > `Pods/` -- Pods not checked in
+
+1. Add, commit and push Pod files
+
+## Jenkins
+
+1. In Xcode, click on the project target icon next to the play button, then click "Manage Schemes...". Click the checkbox to set your project scheme to `Shared`. This is required for Jenkins
+
+1. Add, commit and push the changes.
+
+1. Setup Github webhooks
+    - For pull requests, add:
+        ```
+        https://ci.intrepid.io/ghprbhook/
+        ```
+        Then, select these individual events as triggers:
+        - Issue comment
+        - Pull request
+
+    - For deploying builds, add
+        ```
+        https://ci.intrepid.io/git/notifyCommit?url=https://github.com/IntrepidPursuits/<github-repo-name>&branches=<branches>,<to>,<build>
+        ```
+        Replace the appropriate sections, then select "Just the `push` event" as the webhook trigger
+
+1. Setup Jenkins jobs
+    - Copy `ios-template` and replace the variables as directed. Add the "Record Master Coverage" post-build step to get code coverage comparisons.
+    - Copy `ios-pull-request-template` on Jenkins and replace the variables as directed. Go to Build Triggers and click Advanced, then add "retest this please" as the "Trigger phrase" so that Jenkins will rebuild the job if you add the trigger phrase as a comment on a PR.
```

Comment:
"retest this please" is already globally configured, you only need to edit this field if you want to change the phrase.